builtin/providers/aws/resource_aws_eip.go

```diff
@@ -30,13 +30,13 @@ func resourceAwsEip() *schema.Resource {
 			"instance": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
-                                Computed: true,
+				Computed: true,
 			},
 
 			"network_interface": &schema.Schema{
-                                Type:     schema.TypeString,
-                                Optional: true,
-                                Computed: true,
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Can we do this whitespace diff separately? Sorry to nit - I'm a big `git blame` spelunker :grinning: